clarify the execution contexts of hook handlers with different action (#23699)
kubernetes/website: content/en/docs/concepts/containers/container-lifecycle-hooks.md

### Hook handler execution

When a Container lifecycle management hook is called,

> the Kubernetes management system will execute handler according to the hook action,

Reviewer: Use [present tense](https://kubernetes.io/docs/contribute/style/style-guide/#use-present-tense):

```suggestion
the Kubernetes management system executes the handler according to the hook action,
```

> `exec` and `tcpSocket` will be executed in the container, and `httpGet` will be executed by the kubelet process.

Reviewer: Use [present tense](https://kubernetes.io/docs/contribute/style/style-guide/#use-present-tense):

```suggestion
`exec` and `tcpSocket` are executed in the container, and `httpGet` is executed by the kubelet process.
```

Author: Modify according to [present tense](https://kubernetes.io/docs/contribute/style/style-guide/#use-present-tense). 
Thank you for your advice,please review again.
